helmfile/charts/notify-celery/templates/deployment.yaml

```diff
@@ -0,0 +1,101 @@
+{{- range $node := .Values.nodes.primaryAndScalable }}
```

Comment:
Ah I see - nice!

Reply:
I'm pushing those configs to the nodes section so they can be changed if you like :)